types/src/rsmt2d.rs

```diff
@@ -237,6 +237,19 @@ impl ExtendedDataSquare {
         Ok(eds)
     }
 
+    /// Crate a new EDS that represents an empty block
+    pub fn empty() -> ExtendedDataSquare {
```

Comment:
I wasn't sure which is better, `empty` or `empty_block`

Reply:
I like empty